TLM/TLM/Manager/AbstractGeometryObservingManager.cs

```diff
@@ -64,13 +65,13 @@ public abstract class AbstractGeometryObservingManager : AbstractCustomManager,
                 ExtSegment seg = (ExtSegment)update.segment;
                 if (!seg.valid) {
 #if DEBUGGEO
```

Comment:
Seems odd - `DEBUGGEO` - maybe some old way of debug switching that existed prior to the debug switches?

Reply:
I think that's from the times when Victor tested geometry observer (notify on change etc.).

BTW, I've noticed that FullDebug output folder had name `DebugGeometry` (before Harmony)

Reply:
Yes, I think debug switches were introduced long time after geometry calculation

Reply:
we should replace "DEBUGGEO" with "DEBUG" (or remove completely if possible) at some point

Reply:
Renamed all `#if DEBUGGEO` to `#if DEBUG` and removed `DEBUGGEO` from project definitions